frontend/app-development/layout/AppBar/AppBar.tsx

```diff
@@ -33,6 +33,10 @@ export const AppBar = ({ activeSubHeaderSelection, showSubMenu }: IAppBarProps)
   };
   const { data: user } = useUserQuery();
 
+  const handlePreviewClick = () => {
+    window.location.href = previewPath(org, app);
```

Comment:
Ville det vært bedre å åpne den i et nytt vindu? Tror dette skal gjøre susen:
```suggestion
    window.open(previewPath(org, app), "_blank");
```

Reply:
Vet ikke 🤔 Har du noen innspill på det @nkylstad ? Om vi går for det føler jeg hvertfall hot reload burde være fokus

Reply:
Burde vel egt lande det i første utkast av https://github.com/Altinn/altinn-studio/issues/10227. Det er vel et _bare minimum_ å hvertfall komme seg tilbake 😅

Reply:
Eller det er kanskje dette som er målbildet?
![Screenshot 2023-04-26 at 08 21 55](https://user-images.githubusercontent.com/71079896/234486949-d1465b84-b9e3-41b9-8247-85284d62a870.png)